server/helpers/procBuilder.go

```diff
@@ -31,10 +31,11 @@ import (
 	"github.com/woodpecker-ci/woodpecker/cncd/pipeline/pipeline/frontend/yaml/matrix"
 	"github.com/woodpecker-ci/woodpecker/model"
 	"github.com/woodpecker-ci/woodpecker/remote"
+	"github.com/woodpecker-ci/woodpecker/server"
 )
 
-// Takes the hook data and the yaml and returns in internal data model
-type procBuilder struct {
+// ProcBuilder Takes the hook data and the yaml and returns in helpers data model
```

Comment:
```suggestion
// ProcBuilder Takes the hook data and the yaml and returns in internal data model
```
Oops. I tried internal first then went with helpers, but renaming internal to helpers was a little too aggressive.